lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -1231,6 +1231,9 @@ def install_puppet_agent_pe_promoted_repo_on( hosts, opts )
             scp_to host, File.join(copy_dir_local, download_file), onhost_copy_base
 
             case variant
+            when /^(fedora-22)$/
+              on host, "tar -zxvf #{onhost_copied_download} -C #{onhost_copy_base}"
```

Comment:
all these tars for non-windows should/could probably be pulled out of the case statement